routing/directions_engine.hpp

```diff
@@ -19,8 +20,7 @@ class IDirectionsEngine
 
   virtual void Generate(RoadGraphBase const & graph, vector<Junction> const & path,
                         Route::TTimes & times, Route::TTurns & turns,
-                        vector<Junction> & routeGeometry,
-                        vector<traffic::TrafficInfo::RoadSegmentId> & trafficSegs,
+                        vector<Junction> & routeGeometry, vector<Segment> & trafficSegs,
```

Comment:
As you modifying this API, could you please move cancellable before out parameters (times, turns, routeGeometry and trafficSegs)? According to Google C++ style guide, out params should be last.